src/libstd/rand/os.rs

```diff
@@ -32,7 +32,8 @@ mod imp {
               any(target_arch = "x86_64",
                   target_arch = "x86",
                   target_arch = "arm",
-                  target_arch = "aarch64")))]
+                  target_arch = "aarch64",
+                  target_arch = "powerpc")))]
```

Comment:
Hm, this looks like... all architectures? Perhaps we could just list use linux here?